specification/ai/data-plane/Azure.AI.Projects/preview/2024-07-01-preview/azure-ai-projects.json

```diff
@@ -40,7 +40,7 @@
       {
         "name": "projectName",
         "in": "path",
-        "description": "The Azure AI Studio project name.",
+        "description": "The Azure AI Foundry project name.",
```

Comment:
Please rebase. I already made the renaming changes Studio -> Foundry

Reply:
It seems, it was changed in the project, but did not propagate to swagger:
https://github.com/Azure/azure-rest-api-specs/blob/e0249a61a6b884891b68d0224350624378b26f76/specification/ai/data-plane/Azure.AI.Projects/preview/2024-07-01-preview/azure-ai-projects.json#L43C38-L43C45